feature/agenda/presentation/src/main/java/com/sergius/agenda/presentation/EditTextState.kt

```diff
@@ -0,0 +1,9 @@
+package com.sergius.agenda.presentation
+
+import androidx.compose.foundation.text.input.TextFieldState
+
+data class EditTextState(
+    val initialText: String,
```

Comment:
Does this need to be a separate property or could it be enough to initialize the `TextFieldState` directly with the initial text from the VM constructor? 
My main point is that the initial text isn't state, since it can't change.